src/components/picker/PickerItemsList.tsx

```diff
@@ -66,7 +67,15 @@ const PickerItemsList = (props: PickerItemsListProps) => {
   },
   [children]);
 
+  const renderPropItems = useCallback(({item}: ListRenderItemInfo<PickerItemProps>) => {
+    return <PickerItem key={item.value} value={item.value} label={item.label} disabled={item.disabled}/>;
+  },
+  [items]);
+
   const renderList = () => {
+    if (items) {
```

Comment:
I don't really like adding another return, but I can't say I can think of a better alternative.